openpype/tools/workfiles/save_as_dialog.py

```diff
@@ -218,7 +219,15 @@ def __init__(
 
         # Version number input
         version_input = QtWidgets.QSpinBox(version_widget)
-        version_input.setMinimum(1)
+        version_input.setMinimum(
+            version_start.get_versioning_start(
+                self.data["project"]["name"],
+                self.host.name,
```

Comment:
Here is small issue that not all hosts have available `name` attribute yet. This is one of the places where `os.environ["AVALON_APP"]` should be used because there is not other option.